### PR TITLE
add verbose flag to qvp code

### PR DESCRIPTION
**Pull request by dopplerchase**
While seeing all available radar tilts for each file is useful, I think it is unnecessary to have it print out every single time. So I have added a bool to the input where you can turn that print statement on-off, but default it is off. Feel free to switch it to default on. I just wanted control to turn it off so my progress bar worked properly.

### CODE REVIEW

**Comment by zssherman** [comment]
Thanks @dopplerchase for the PR! I agree this is a better way to handle the print statements. Only recommendation, can we remove the second sentence of the docstring for the verbose parameter? Besides that once the tests run and pass should be good.

File: pyart/retrieve/qvp.py [hunk 1]
```diff
@@ -25,6 +25,9 @@ def quasi_vertical_profile(radar, desired_angle=None, fields=None, gatefilter=No
     desired_angle : float
         Radar tilt angle to use for indexing radar field data.
         None will result in wanted_angle = 20.0
+    verbose : bool
+        boolean flag to turn the printing of radar tilts on or off. 
+        Can be very annoying if you process alot of files.
```
Comment (mgrover1) [inline pyart/retrieve/qvp.py:30]: Yeah - I would agree with @zssherman here, can you remove this line? The rest looks great 😁

**Review by mgrover1 — CHANGES_REQUESTED** [review]
One small comment - other than that, looks great!

File: pyart/retrieve/qvp.py (hunk 1 above)
Comment (dopplerchase) [reply pyart/retrieve/qvp.py:30]: Git novice over here. How do I make a change to an exisitng pull request?

Comment (zssherman) [reply pyart/retrieve/qvp.py:30]: or to your main

Comment (mgrover1) [reply pyart/retrieve/qvp.py:30]: You'll want to make this change on your branch, commit, then push again

**Review by mgrover1 — APPROVED** [review]
Thanks for making those changes @dopplerchase ! Once the tests pass, this looks good to merge 🚀

**Pull request merged**